### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by popeinabar**

### CODE REVIEW

**Review by PatelNeelMahesh — APPROVED** [review]
Keep updating the readme file as you add more features,
Give description for PR.
Give meaningful commit message.

You can merge pull request.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "react-apps",
```
Comment (aditya-190) [inline package.json:2]: Pls replace `name` with your `project-name` i.e. - `brickel`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
```
Comment (aditya-190) [inline package.json:6]: testing libraries can be added in `devDependencies` or you can even remove them as you will be not using them for now.

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (aditya-190) [inline public/index.html:27]: Update title to be your `project-name`

File: public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
```
Comment (aditya-190) [inline public/manifest.json:4]: Icons can be removed as they are redundant.

File: public/manifest.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (aditya-190) [inline public/manifest.json:3]: Add your `project-name` here

File: src/About.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  const data={
+    start_journey:'',
+    hero_heading:'About Us',
+    hero_para1:'Hello, and welcome to BRICKEL! My name is AYUSH, and Im the founder of this community-driven platform. Im passionate about education and believe that everyone should have access to high-quality learning resources, regardless of their background or financial situation. Thats why I created BRIKEL to provide a space where people can trade knowledge, skills, and resources for free, and support each other in their learning journey. I hope you enjoy using our platform, and feel free to reach out to me if you have any questions or feedback.',
+    hero_para2:'',
+    image_home:group,
+  }
+  return (
+    <>
+      <NavBar></NavBar>
```
Comment (aditya-190) [inline src/About.js:18]: Replace it with <NavBar />

File: src/App.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+import NavBar from './component/NavBar';
+import User from './User';
+import Login from './Login';
+import Footer from './component/Footer';
+
+function App() {
+  return (
+
+    <>
+    <NavBar></NavBar>
```
Comment (aditya-190) [inline src/App.js:16]: Replace it with <NavBar />

File: src/App.test.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
```
Comment (aditya-190) [inline src/App.test.js:2]: This file can be removed.

File: src/Home.js [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+    hero_heading:'Tutoring',
+    hero_para1:'Teach others to get points and use that points to get free one on one learning...',
+    hero_para2:'Teaching is another form of learning!!!',
+    image_home:abc,
+    page:'home'
+  }
+  return (
+    <>
+      <header className='home-header'>
+        <NavBar></NavBar>
```
Comment (aditya-190) [inline src/Home.js:23]: Why `NavBar` is used inside `Home.js` when you have already defined it before Routes in `App.js` ?

File: src/Learn.js [hunk 10]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Serch from './component/Serch.js'
+
+const Learn = () => {
+  return (
+    <>
+        <div>
+          
+          <Serch/>
```
Comment (aditya-190) [inline src/Learn.js:9]: Please fix the misspelled word - `Search`

File: src/component/HeroSec2.js [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+    <div className='text2'>
+    <h2 className='hero2-heading'>Learn</h2>
+    <p className='hero2-para1'>Learn for free from the points you you can earn by login in and 
+by teaching others....</p>
+    <p className='hero2-para'>The great aim of education is not knowledge but action!!!</p>
+    </div>
+
+    <div className='start2-journey'>
+    <p>Lets goo...</p>
+    <Link to={'./learn'}>
```
Comment (aditya-190) [inline src/component/HeroSec2.js:20]: Please change i to be `/learn`

File: src/data.js [hunk 12]
```diff
@@ -0,0 +1,9 @@
+// import React from 'react'
+
```
Comment (aditya-190) [inline src/data.js:2]: You can delete this file if you are not using this component.

File: src/index.js [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+    <App />
+    </BrowserRouter>
+    
+  </React.StrictMode>
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```
Comment (aditya-190) [inline src/index.js:21]: This can be removed as you will not be using this in your project for now.

**Review by aditya-190 — CHANGES_REQUESTED** [review]
We applaud Ayush for your impressive start!. To ensure that the project continues to meet high standards, a few changes are necessary before merging. Please implement these changes and submit updated PR. We appreciate your hard work and look forward to your continued contributions to the project.

**Pull request merged**